implementations.tex

```diff
@@ -62,6 +62,9 @@ \section{Execution Based} \label{execution_based}
 to 0.
 
 To resume execution, the debug module sets a flag which causes the hart to execute a {\tt dret}.
+{\tt dret} is an instruction that only has meaning while in Debug Mode and
```

Comment:
I think that "only has meaning" loses the illegal instruction exception behavior in non-debug-mode.  Can you copy this sentence down here: "Execution of this instruction outside of Debug Mode causes an illegal instruction exception."

I don't think that the PB execution sentence needs to be copied here because the point of that was to tell debuggers to avoid dret in the PB.  But this "only has meaning" outside the PB sentence basically says the same thing (and that instruction is otherwise illegal and sane debuggers shouldn't be executing illegal instructions anyway).